### DISCUSSION TITLE
Import VISSIM model to CARLA

### DISCUSSION THREAD

**Discussion by hotmedal** [carla-simulator/carla#5501] [Q&A]
Hi, I have very little experience with both CARLA and VISSIM. I would like a high level step-by-step procedure to get a 3D VISSIM model (with all the link information e.g. bicycle and pedestrian only, traffic lights, etc. ) into a CARLA map. I will have to figure out the details by myself but an overview would be greatly appreciated.

**Comment by Ahmad1441** [comment]
Hi Sir, have you figured it out?

**Reply by hotmedal** [reply]
No, I ended up not using VISSIM. Instead, I used openstreetmap and carla's included co-simulation tools with SUMO.
